src/Database/Esqueleto/PostgreSQL.hs

```diff
@@ -41,13 +53,26 @@ random_ = unsafeSqlValue "RANDOM()"
 arrayAgg :: SqlExpr (Value a) -> SqlExpr (Value [a])
 arrayAgg = unsafeSqlFunction "array_agg"
 
+-- | Like 'array_agg' but removes @NULL@ values
+arrayAgg' :: PersistField [a] => SqlExpr (Value (Maybe a)) -> SqlExpr (Value [a])
+arrayAgg' =  arrayRemoveNull . arrayAgg
```

Comment:
A `'` by itself typically means "this value, but modified in some way," where the modification is obvious in the code you're reading. A more self-documenting name (eg `arrayAggNonNull`) would be clearer.

Reply:
@parsonsmatt I tend to agree. I'm okay with prime-markers (apostrophe) as signifying distaff or strictness, but not anything to do with `Maybe`.